packages/trader/src/AppV2/Hooks/useContractsForCompany.ts

```diff
@@ -151,7 +152,7 @@ const useContractsForCompany = () => {
                             ? 'euro_atm'
                             : (contract.barrier_category as TConfig['barrier_category']);
                     config.default_stake = contract.default_stake;
-
+                    if (type === contract_type) setDefaultStake(contract.default_stake);
```

Comment:
default stake for every trade type is coming in `contract_for_` API response. Setting the default stake to the trade store for active (selected) trade type after receiving it